src/js/views/patients/patient/sidebar/README.md

```diff
@@ -2,13 +2,24 @@
 
 ## Defined Widgets
 
-All available widget types are currently located in the [sidebar_views file](https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/views/patients/patient/sidebar/sidebar_views.js#L24)
+All available widget types are currently located in the [sidebar_views file](https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/views/patients/patient/sidebar/sidebar_views.js#L39)
```

Comment:
Let's add a line in the sidebar_views like `// NOTE: These widgets are documented in ./README.md`